### ISSUE TITLE
[Feature Request]: Transcripts 

### ISSUE THREAD

**Issue by Victor239** [FreeTubeApp/FreeTube#3487]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

Video transcripts aren't available within the app. 

### Proposed Solution

Similar UX to YouTube, having a text box with timestamped transcript as well as being able to save it as a text file.

### Alternatives Considered

N/A

### Issue Labels

display more information to user, new feature

**Comment by ChunkyProgrammer** [comment]
Can you provide an example video that has a transcript?

**Comment by absidue** [comment]
@ChunkyProgrammer Every video with subtitles e.g. https://www.youtube.com/watch?v=1YFR20MmvpM

On YouTube on the watch page, click the 3 dots button next to the save one:
![show-transcript](https://user-images.githubusercontent.com/48293849/236407074-1912558e-38a6-425e-860e-55d4467e3c11.png)


Both YouTube.js and Invidious don't currently support it. When I looked into it in the past, it requires messing around with protobufs (to tell it which subtitle track to pick), so definitely something that we want support for upstream first.

**Comment by CCHeckman** [comment]
Definitely, you need a place to get the transcript. That's the only bad thing I've noticed so far. 8-)

**Comment by absidue** [comment]
On YouTube the transcripts are just the captions displayed differently (yes the text content is identical). Until we get around to implementing a special UI for that, you can also click the download button underneath a video and download the captions file, then you can have it open in a text editor next to FreeTube.

**Comment by CCHeckman** [comment]
> you can also click the download button underneath a video and download the captions file, then you can have it open in a text editor next to FreeTube.

Okay, I'll try that. I thought that was to download the video itself. (Downloading the video would also be a handy feature to have.)

**Comment by ExtRIELICi** [comment]
Any news about this?

**Comment by NathanGanesh** [comment]
any update?

**Comment by mrtrollex** [comment]
Hi,
What’s the status of the request? This would be a really useful feature, since nowadays a lot of people use AI to summarize transcripts.
